examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>Perhaps the top-k evaluation using the entire catalog of items can be moved to the end of section <code>Building the Model</code> right after the training of the retrieval model. We can then explain to the user the two evaluation strategies we have in merlin models: </p><ul><li>Compute metrics using the negative sampling strategy used for training (via the retrieval model).</li><li>Evaluate the trained retrieval model using the entire catalog (via the <code>topk_model</code>  )</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>